src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -68,6 +68,13 @@ def __init__(
         elif beta_schedule == "squaredcos_cap_v2":
             # Glide cosine schedule
             self.betas = betas_for_alpha_bar(num_train_timesteps)
+        elif beta_schedule == "sigmoid":
```

Comment:
Very cool! Nice to see that "vanilla" DDPM can be used that easily

Reply:
According the author the model performs better quantitatively with a different scheduler, but visually I noticed no difference.